modules/react-native-status/ios/RCTStatus/RCTStatus.m

```diff
@@ -318,6 +318,61 @@ - (void)handleSignal:(NSString *)signal
     callback(@[result]);
 }
 
+//////////////////////////////////////////////////////////////////// getConnectionStringForBootstrappingAnotherDevice
+RCT_EXPORT_METHOD(getConnectionStringForBootstrappingAnotherDevice:(NSString *)configJSON
+                  callback:(RCTResponseSenderBlock)callback) {
+
+    NSData *configData = [configJSON dataUsingEncoding:NSUTF8StringEncoding];
+    NSDictionary *configDict = [NSJSONSerialization JSONObjectWithData:configData options:NSJSONReadingMutableContainers error:nil];
+    NSLog(@"configDict is ====> %@", configDict);
+    NSString *keyUID = [configDict objectForKey:@"keyUID"];
+    NSLog(@"keyUID is ====> %@", keyUID);
+//    NSString *keystoreDir = [@"/keystore/" stringByAppendingString:keyUID];
```

Comment:
commented code & logs. Probably should clean this up.

Reply:
kept this code because I may or may not want to use it in the future but will definitely clean this up in the follow up PRs